webapp/src/Controller/Jury/ContestController.php

```diff
@@ -492,6 +492,21 @@ public function editAction(Request $request, int $contestId): Response
         $form->handleRequest($request);
 
         if ($form->isSubmitted() && $form->isValid()) {
+            $formData = $form->getData();
+            $timeZones = [];
+            foreach(['Activate','Deactivate','Start','End','Freeze','Unfreeze'] as $timeString) {
+                $fields = explode(' ', $formData->{'get'.$timeString.'timeString'}());
+                if(count($fields)>1) {
+                    $timeZones[] = $fields[2];
+                }
+            }
+            if(count(array_unique($timeZones))>1) {
```

Comment:
Could we try to format this according to the rest of the file (PSR)?

Reply:
```suggestion
            if (count(array_unique($timeZones))>1) {
```
So this?

Reply:
No, even `if (count(array_unique($timeZones)) > 1) {`